packages/core/useSwipe/index.ts

```diff
@@ -15,21 +16,22 @@ export enum SwipeDirection {
  * @param options
  */
 export function useSwipe(
-  target: MaybeRef<HTMLElement | null | undefined>,
-  options?: {
+  target: MaybeRef<EventTarget>,
+  options: {
     threshold?: number
+    preventScrolling: boolean
     onSwipe?: (e: TouchEvent) => void
     onSwipeEnd?: (e: TouchEvent, direction: SwipeDirection) => void
-  },
+  } = { preventScrolling: false },
```

Comment:
```suggestion
  } = {},
```

it will be overridden when providing any of the options, should do it in the spreading below